File: trip_parser/v3/post/Python SDK/trip_parser.py
import json
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
  with open('../request_body.json') as file:
    body = json.load(file)
    response = amadeus.travel.trip_parser.post(body)
    print(response.data)
except ResponseError as error:
    raise error

File: trip_parser/v3/post/Node SDK/trip_parser.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

//Read trip data from file
const body = require('../request_body.json');
amadeus.travel.tripParser.post(JSON.stringify(body))
  .then(response => console.log(response))
  .catch(err => console.error(err));

/**
 * If you don't have your base64 encoded payload, you can use
 * the SDK helper method to create your request body
 * '''js
 * const base64Payload = amadeus.tripParser.fromFile(fileInUTF8Format)
 * '''
 */